packages/pds-ember/tests/dummy/app/styles/app.scss

```diff
@@ -109,3 +110,22 @@
     width: auto !important;
   }
 }
+
+
+// SHAME
```

Comment:
```suggestion
```

If this was left in intentionally for some reason, please provide additional context.